cypress/e2e/seo.cy.ts

```diff
@@ -0,0 +1,43 @@
+const pages = [
```

Comment:
Maybe we can move this const into an enum file called PageType. Then we can use them throuh out the code base instead of just this file.